docker/compose_files/files/mysqlconf/mysql.cnf

```diff
@@ -0,0 +1,9 @@
+[client]
```

Comment:
Do we need this to work?  We have to be careful, because on prod, the mysql server is setup not through docker but is on another server.

Reply:
Added accidentally, will remove